cluster/examples/wordpress/aws/wordpress.yaml

```diff
@@ -21,8 +21,7 @@ spec:
   engine: mysql
   masterUsername: masteruser
   providerRef:
-    name: demo-aws-provider
-    namespace: crossplane-system
+    name: example
   writeConnectionSecretToRef:
     name: demo-rds
```

Comment:
I think we need to add `namespace` here.

Reply:
Good catch! Thank you. I wonder how I was able to submit this example successfully without that. 🤔

Reply:
Ah - it has commented out security groups so I must have assumed this example did not apply because of that.